api/rest/stat_series.md

```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
 - `unit`: The dcid of the preferred `unit` value.
 - `scaling_factor`: The preferred `scalingFactor` value.
 
-## GET Request
+## How to construct a request to the place statistics time series endpoint
 
-**Examples**
+### Step 1: Assembling the information you will need
 
-```bash
-curl 'https://api.datacommons.org/stat/series?place=geoId/06&stat_var=Count_Person_Male'
-```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
```

Comment:
Maybe just "This endpoint requires two arguments:" and leave the additional optional as you have now "In addition to these required properties, this endpoint also.."

Reply:
In general, I think it's best not to enumerate the arguments for maintainability